### DISCUSSION TITLE
/.well-known/openid-configuration shows HTTP not HTTPS

### DISCUSSION THREAD

**Discussion by StephanMeijer** [fief-dev/fief#108] [Q&A]
![image](https://user-images.githubusercontent.com/3169264/200125961-0ea25658-d1dd-4855-ae09-f4c8142d9a1e.png)
How to get these URL's to be https, not http?

**Comment by fief-bailiff** [comment]
Hail, @StephanMeijer 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
So, you are currently running your own instance on your local machine? Is it hosted on something like http://localhost:8000?

**Reply by StephanMeijer** [reply]
Yes, running my own instance in a production environment. Using Traefik for SSL.

**Reply by StephanMeijer** [reply]
Running Fief in Docker.

**Reply by frankie567** [reply]
Ok! Well in this case, you need to make sure of two things:

* Your workspace domain should match your domain. If you serve it e.g. behind `fief.mydomain.com`, you should have `fief.mydomain.com` in the `domain` column of the `workspaces` table.
* Since you're behind a proxy, you need to set the environment variable `FORWARDED_ALLOW_IPS` to `*`. It tells the server to trust the SSL headers of Traefik.

**Reply by StephanMeijer** [reply]
Workspace domain; is that an env var? Or can I set that somewhere in the UI?

**Reply by frankie567** [reply]
Yes, it corresponds to `ROOT_DOMAIN` and `FIEF_DOMAIN`:

* `ROOT_DOMAIN` should be `mydomain.com`
* `FIEF_DOMAIN` should be `fief.mydomain.com`

**Reply by StephanMeijer** [reply]
Yes, I set those correcrly. Will try `FORWARDED_ALLOW_IPS`

**Reply by StephanMeijer** [reply]
Like a charm! thanks!  It was FORWARDED_ALLOW_IPS

Is it smart to set this to * or should I set it to a docker container host?

**Reply by frankie567** [reply]
Normally, set it to `*` should be okay if the Fief container is not accessible from the internet.

If you really want to limit the IP, you should get the internal Docker IP of your Traefik container and set this one.

Ref: https://stackoverflow.com/questions/66683136/how-to-determine-forwarded-allow-ips-for-uvicorn-server-from-docker-network
